packages/database/src/migrations/20210907222655-LESMISPrimaryEducationCohortSurvivalRate-modifies-data.js

```diff
@@ -0,0 +1,423 @@
+    );
+    await insertObject(
+      db,
+      'indicator',
+      generateFiveYearIndicator(
+        `csr_${baseCode}_summary`,
+        `csr_${baseCode}_average`,
+        `csr_${baseCode}_target`,
+      ),
+    );
+  }
+};
+
+const deleteAllIndicators = async db => {
+  for (const { baseCode } of INDICATOR_CONFIGS) {
+    await deleteObject(db, 'indicator', { code: `csr_${baseCode}_summary` });
+    await deleteObject(db, 'indicator', { code: `csr_${baseCode}_average` });
+    await deleteObject(db, 'indicator', { code: `csr_${baseCode}_target` });
+  }
+};
+
+const DASHBOARD_ITEMS = [
+  {
+    code: 'LESMIS_cohort_survival_rate_national',
+    reportConfig: DRILL_DOWN_REPORT_CONFIG,
+    frontEndConfig: DRILL_DOWN_FRONT_END_CONFIG,
+  },
+  {
+    code: 'LESMIS_ESSDP_PE_HLO1',
+    reportConfig: LIST_REPORT_CONFIG,
+    frontEndConfig: LIST_FRONT_END_CONFIG,
+  },
+];
+
+// Same util functions as always
+const addNewDashboardItemAndReport = async (
+  db,
+  { code, frontEndConfig, reportConfig, permissionGroup, dashboardCode, entityTypes, projectCodes },
+) => {
+  // insert report
+  const reportId = generateId();
+  const permissionGroupId = (
+    await findSingleRecord(db, 'permission_group', { name: permissionGroup })
+  ).id;
+  await insertObject(db, 'report', {
+    id: reportId,
+    code,
+    config: reportConfig,
+    permission_group_id: permissionGroupId,
+  });
+
+  // insert dashboard item
+  const dashboardItemId = generateId();
+  await insertObject(db, 'dashboard_item', {
+    id: dashboardItemId,
+    code,
+    config: frontEndConfig,
+    report_code: code,
+  });
+
+  // insert relation record connecting dashboard item to dashboard
+  const dashboardId = (await findSingleRecord(db, 'dashboard', { code: dashboardCode })).id;
+  const maxSortOrder = (
+    await findSingleRecordBySql(
+      db,
+      `SELECT max(sort_order) as max_sort_order FROM dashboard_relation WHERE dashboard_id = '${dashboardId}';`,
+    )
+  ).max_sort_order;
+  await insertObject(db, 'dashboard_relation', {
+    id: generateId(),
+    dashboard_id: dashboardId,
+    child_id: dashboardItemId,
+    entity_types: `{${entityTypes.join(', ')}}`,
+    project_codes: `{${projectCodes.join(', ')}}`,
+    permission_groups: `{${permissionGroup}}`,
+    sort_order: maxSortOrder + 1,
+  });
+};
+
+const removeDashboardItemAndReport = async (db, code) => {
+  await db.runSql(`DELETE FROM dashboard_item WHERE code = '${code}';`); // delete cascades to dashboard_relation
+  await db.runSql(`DELETE FROM report WHERE code = '${code}';`);
+};
+
+exports.up = async function (db) {
+  await generateAllIndicators(db);
+  for (const { code, frontEndConfig, reportConfig } of DASHBOARD_ITEMS) {
+    await addNewDashboardItemAndReport(db, {
+      code,
+      frontEndConfig,
+      reportConfig,
+      permissionGroup: 'LESMIS Public',
+      dashboardCode: 'LESMIS_ESSDP_PrimarySubSector_HLO1',
```

Comment:
Just realised that I only actually added this dashboard in WAI-44 so while it was working on my local it actually relies on [another PR](https://github.com/beyondessential/tupaia/pull/3171)

I'm just going to wait until that PR gets merged so I don't have to do complicated git stuff